pkg/test/util/policy/policy.go

```diff
@@ -43,9 +43,16 @@ func (p TestPolicy) TearDown() {
 	}
 }
 
+// ApplyPolicyFile applies a policy file from testdata directory of the test.
 func ApplyPolicyFile(t *testing.T, env *kube.Environment, namespace string, fileName string) *TestPolicy {
+	joinedPath := path.Join(testDataDir, fileName)
+	return ApplyPolicyAnyFilePath(t, env, namespace, joinedPath)
+}
+
+// ApplyPolicyAnyFilePath runs `kubectl -f <namespace> <fileName>` where fileName is the real/explicit path of the file.
+func ApplyPolicyAnyFilePath(t *testing.T, env *kube.Environment, namespace string, fileName string) *TestPolicy {
```

Comment:
What does `Any` mean here?

Reply:
`ApplyPolicyFile` in `policy.go` assumes the yaml configs are under `testdata`. Since our RBAC yaml files using template for namespaces, it's better to put them in the tmp directory created by the test. So `AnyFilePath` means the file can be anywhere. I can change it to `ApplyPolicyAbsoluteFilePath` to make it more clear, wdyt?

Reply:
Okay, I think `ApplyPolicyFromFilePath` might be better in this case.